test/functional/wallet_bumpfee.py

```diff
@@ -668,5 +670,30 @@ def test_no_more_inputs_fails(self, rbf_node, dest_address):
     self.clear_mempool()
 
 
+def test_feerate_checks_replaced_outputs(self, rbf_node):
```

Comment:
If I understand this test right, it checks whether the absolute fee of a smaller replacement transaction sufficiently surpasses that of a larger original transaction.

I looked around in `wallet_bumpfee.py`, and had the impression that we generally seem to assume that the replacement is the same size as the original. If we don’t check it yet, we should perhaps also add a test case where a larger replacement transaction supersedes a smaller original’s feerate sufficiently (although I might have overlooked that test case).

Reply:
> If I understand this test right, it checks whether the absolute fee of a smaller replacement transaction sufficiently surpasses that of a larger original transaction.

No, it doesn't check any feerates since we can't see the candidate bump transaction. The calculations here are just for figuring out a feerate that would fail the modified checks and checking that `bumpfee` fails as expected.

> I looked around in `wallet_bumpfee.py`, and had the impression that we generally seem to assume that the replacement is the same size as the original. If we don’t check it yet, we should perhaps also add a test case where a larger replacement transaction supersedes a smaller original’s feerate sufficiently (although I might have overlooked that test case).

There are no size checks since we are not making specific bump transactions to test replacement behavior. Rather we checking whether the `bumpfee` command succeeds.